index.html

```diff
@@ -0,0 +1,177 @@
+      <ul class="socials">
+        <li><a href=""><img src="./images/socials/facebook.svg" alt="Facebook icon"></a></li>
+        <li><a href=""><img src="./images/socials/twitter.svg" alt="Twitter icon"></a></li>
+        <li><a href="#">English</a></li>
+        <li><a href="#">My Page</a></li>
+        <li><a href="#">Logout</a></li>
+      </ul>
+
+      <div>
+        <a href="./index.html">
+          <img src="./images/tech-splendor.png" alt="Project Logo" class="desktop-project-logo">
+        </a>
+        <ul class="menu desktop-menu">
+          <li><a href="./about.html">About</a></li>
+          <li><a href="#">Program</a></li>
+          <li><a href="#">Join</a></li>
+          <li><a href="#">Sponsor</a></li>
+          <li><a href="#">News</a></li>
+          <li><a href="#">TS Campaign</a></li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+
+  <section class="hero">
+    <q>Hello! Tech World!</q>
+    <h1>TECHSPLENDOR SYMPOSIUM 2023</h1>
+    <p class="intro">
+      TechSplendor, the pinnacle of technological excellence! Prepare to immerse yourself in an extraordinary tech
+      journey like never before.
+    </p>
+    <p class="schedule">2023.10.16(MON) ~ 23rd(FRI)</p>
+    <p class="venue">
+      @ Mayor's Garden of Lira, Coronation Park and more
+    </p>
+  </section>
+
+  <main>
+    <section class="main-program">
+      <h2>Main Program</h2>
+      <hr>
+      <div class="programs">
+        <article>
+          <img src="./images/main/program-1.png" alt="Picture of first program">
+          <h3>Expert Talks</h3>
+          <p>Listen to experts, be inspired by their journeys and learn from their invaluable experiences.</p>
+        </article>
+        <article>
+          <img src="./images/main/program-2.png" alt="Picture of second program">
+          <h3>Interactive Workshops</h3>
+          <p>Dive into hands-on workshops that cater to all skill levels, from coding novices to seasoned developers.
+          </p>
+        </article>
+        <article>
+          <img src="./images/main/program-3.png" alt="Picture of third program">
+          <h3>Challenges & Competitions</h3>
+          <p>Put your skills to the test in exhilarating tech challenges and competitions</p>
+        </article>
+        <article>
+          <img src="./images/main/program-4.png" alt="Picture of fourth program">
+          <h3>Innovation Showcases</h3>
+          <p>Witness the future come to life in our innovation showcases by cutting-edge startups and tech companies</p>
+        </article>
+        <article>
+          <img src="./images/main/program-5.png" alt="Picture of fifth program">
+          <h3>Networking - Collaboration</h3>
+          <p>Connect with fellow tech enthusiasts, entrepreneurs, and industry experts.</p>
+        </article>
+      </div>
+      <a href="#" class="join">Join TechSplendor 2023</a>
+      <a href="#" class="see-all">SEE THE WHOLE PROGRAM</a>
+    </section>
+
+    <section class="experts">
```

Comment:
- [ ] [REQUIRED] Good job so far! It would be nice if you decrease the size of  `horizontal line` located under the `heading` and create `small gray horizontal line` between to `expertise:` and ` affiliation:` texts. Finally implement  `chess shape` to each avatar's background 🙏🏼 

![image](https://github.com/dd-obua/Mic-Portal-Captone/assets/63915024/de00eb56-9d5d-4d55-9f0c-4c5f030437a8)